client/helpers/formatTime.js

```diff
@@ -14,10 +14,13 @@ const formatAsEliteDateTime = (timestamp) => moment(timestamp).add(ELITE_GAME_YE
 
 const formatAsEliteDate = (timestamp) => moment(timestamp).add(ELITE_GAME_YEAR_DESPARITY, 'years').format('DD MMM YYYY').toUpperCase()
 
+const formatAsEliteDateLong = (timestamp) => moment(timestamp).add(ELITE_GAME_YEAR_DESPARITY, 'years').format('DD MMMM YYYY')
+
 
 
 export {
   // eslint-disable-next-line import/prefer-default-export
```

Comment:
Since we have more than one export from this file now, we can remove this config override.